OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -363,4 +355,11 @@
     <Message Text="DEBUG TargetName:             $(TargetName)"/>
     -->
   </Target>
-</Project>
+  <PropertyGroup>
+    <PostBuildEvent>IF  EXIST  "%25VS140COMNTOOLS%25"  CALL  "%25VS140COMNTOOLS%25vsvars32.bat"
```

Comment:
@Phrohdoh: you just tidied up our existing postbuild tasks, so do you have any suggestions on how to improve on this mess?